docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -617,6 +617,7 @@ Please see xref:native-and-ssl.adoc#working-with-containers[our Using SSL With N
 
 NOTE: To use Mandrel instead of GraalVM CE, update the `FROM` clause to: `FROM quay.io/quarkus/ubi-quarkus-mandrel-builder-image:$TAG AS build`.
 `$TAG` can be found on the https://quay.io/repository/quarkus/ubi-quarkus-mandrel-builder-image?tab=tags[Quarkus Mandrel Images Tags page].
+Note that, starting 22.3, Mandrel does not produce a `-java-11` variant. Use the `-java-17` image instead.
```

Comment:
Shouldn't both of these notes be wrapped in

```
[NOTE]
====
====
```
?

e.g.
```suggestion
[NOTE]
====
To use Mandrel instead of GraalVM CE, update the `FROM` clause to: `FROM quay.io/quarkus/ubi-quarkus-mandrel-builder-image:$TAG AS build`.
`$TAG` can be found on the https://quay.io/repository/quarkus/ubi-quarkus-mandrel-builder-image?tab=tags[Quarkus Mandrel Images Tags page].
====

[NOTE]
====
Starting with 22.3, Mandrel does not provide a `-java11` variant anymore. Use the `-java17` image instead.
====
```